src/components/v5/common/ActionSidebar/partials/forms/ManageVerifiedMembersForm/consts.ts

```diff
@@ -0,0 +1,41 @@
+import { InferType, object, string } from 'yup';
+
+import { MAX_ANNOTATION_LENGTH } from '~constants';
+import { formatText } from '~utils/intl';
+import { ACTION_BASE_VALIDATION_SCHEMA } from '~v5/common/ActionSidebar/consts';
+import { CardSelectOption } from '~v5/common/Fields/CardSelect/types';
+
+export enum AddRemoveMembersType {
+  Add = 0,
+  Remove = 1,
+}
+
+export const getAddOrRemoveMembersOptions = (): {
+  addRemoveMembersOptions: CardSelectOption<AddRemoveMembersType>[];
+} => {
+  return {
+    addRemoveMembersOptions: [
+      {
+        label: formatText({ id: 'actionSidebar.option.addMembers' }),
+        value: AddRemoveMembersType.Add,
+      },
+      {
+        label: formatText({ id: 'actionSidebar.option.removeMembers' }),
+        value: AddRemoveMembersType.Remove,
+      },
+    ],
+  };
+};
+
+export const validationSchema = object()
+  .shape({
+    createdIn: string().defined(),
```

Comment:
Beware that if this PR: #1702 gets merged first then this should be updated to `number`. Otherwise, I'll update this field there if this one gets merged first.

Reply:
That other PR was merged so this should be updated to a `number` schema :)

Reply:
@ArmandoGraterol Changed to number :) Also this `The upper corners of the verified table seem to be cut-off:` is fixed in other PR